DRODLib/TemporalClone.h

```diff
@@ -58,6 +58,7 @@ class CTemporalClone : public CMimic
 
 	//player stats on creation
 	UINT wIdentity;
+	UINT wAppearance;
```

Comment:
Wow!  This change is this simple?  I didn't realize that.  Amazing.

Suggest initializing wAppearance in the constructor method (to a sentinel value) so that it will be apparent if it's not being set properly anywhere during use in the future.